CoreJS build fails during deploy

I've found a strange bug when using the deploy script with CI:

1. Start with clean repository (corejs not build ...)
2. Run `npm install` and `node deploy.js meteor.com`
3. CoreJS build fails:

```bash
Building core-js@1.2.1 without ES6 number constructor...

> meteor-webpack-react@0.0.1 build-core-js /home/janek/tmp/meteor-webpack-react-jbbr
> grunt --gruntfile node_modules/core-js/Gruntfile.js build:es5,es6,es7,js,web --blacklist=es6.number.constructor --path=core-js-no-number

Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module 'LiveScript'
Warning: Task "build:es5,es6,es7,js,web" not found. Use --force to continue.

Aborted due to warnings.

npm ERR! Linux 4.2.2-1-ARCH
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "build-core-js"
npm ERR! node v4.1.1
npm ERR! npm  v3.3.5
npm ERR! code ELIFECYCLE
npm ERR! meteor-webpack-react@0.0.1 build-core-js: `grunt --gruntfile node_modules/core-js/Gruntfile.js build:es5,es6,es7,js,web --blacklist=es6.number.constructor --path=core-js-no-number`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the meteor-webpack-react@0.0.1 build-core-js script 'grunt --gruntfile node_modules/core-js/Gruntfile.js build:es5,es6,es7,js,web --blacklist=es6.number.constructor --path=core-js-no-number'.
npm ERR! This is most likely a problem with the meteor-webpack-react package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt --gruntfile node_modules/core-js/Gruntfile.js build:es5,es6,es7,js,web --blacklist=es6.number.constructor --path=core-js-no-number
npm ERR! You can get their info via:
npm ERR!     npm owner ls meteor-webpack-react
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/janek/tmp/meteor-webpack-react-jbbr/npm-debug.log
mv: no such file or directory: core-js-no-number.js

ln: Source file does not exist

Hash: a941a9b3171a0f8f2a31  
Version: webpack 1.12.2
Time: 733ms
```


After using the `dev.js` script at least one time `deploy.js` runs fine.

I don't know why this happens as core-js build script is required the same way. Maybe it's due to different environment variables?

--- a/predeploy.js
+++ b/predeploy.js
@@ -23,12 +23,12 @@ var loadClientBundleHtml = path.join(dirs.webpack, 'loadClientBundle.html');
 var loadClientBundleLink = path.join(dirs.meteor, 'client/loadClientBundle.html');
 var requireServerBundleJs = path.join(dirs.meteor, 'server/require.server.bundle.js');
 
+require('./core-js-custom-build');
+
 module.exports = function(callback) {
   if (!process.env.NODE_ENV) {
     process.env.NODE_ENV = env.NODE_ENV = 'production';
   }
-
-  require('./core-js-custom-build');
 
   if (fs.existsSync(loadClientBundleLink)) rm(loadClientBundleLink);
   if (fs.existsSync(requireServerBundleJs)) rm(requireServerBundleJs);

--- a/predeploy.js
+++ b/predeploy.js
@@ -23,9 +23,9 @@ var loadClientBundleHtml = path.join(dirs.webpack, 'loadClientBundle.html');
 var loadClientBundleLink = path.join(dirs.meteor, 'client/loadClientBundle.html');
 var requireServerBundleJs = path.join(dirs.meteor, 'server/require.server.bundle.js');
 
-require('./core-js-custom-build');
-
 module.exports = function(callback) {
+  require('./core-js-custom-build');
+
   if (!process.env.NODE_ENV) {
     process.env.NODE_ENV = env.NODE_ENV = 'production';
   }

